packages/wonder-blocks-breadcrumb/docs.md

```diff
@@ -0,0 +1 @@
+A breadcrumb trail consists of a list of links to the parent pages of the current page in hierarchical order. It helps users find their place within a website or web application.
```

Comment:
I think just moving this into the comment for Breadcrumb will achieve what I mentioned above.